02-eda.R

```diff
@@ -32,21 +32,19 @@ max_val_int <- collatz_df %>%
 #for even starting integers compared to odd ones?
   
 
-even_odd_avg_len <- collatz_df %>%
+even_odd_avg <- collatz_df %>%
                        group_by(parity) %>% 
                        summarise(avg = mean(length)) %>%
-                       select(avg) %>%
-                       unlist()
-
-even_odd_avg_len[2] / even_odd_avg_len[1]
+                       select(avg)
+  
+even_odd_avg_len <- even_odd_avg$avg  
 
-even_odd_sd_len <- collatz_df %>%
+even_odd_sd <- collatz_df %>%
                     group_by(parity) %>%
                     summarise(sd = sd(length)) %>%
-                    select(sd) %>%
-                    unlist()
+                    select(sd)
 
-even_odd_sd_len[2] / even_odd_sd_len[1]
+even_odd_sd_len <- even_odd_sd$sd
```

Comment:
Great Job! Here another suggestion, you may use ```summarise()``` instead the longer version you've done.